src/components/plugin/plugin-app.tsx

```diff
@@ -314,7 +321,9 @@ export default class PluginApp extends React.Component<IProps, IState> {
         content: container,
         title: `Term: ${word}`,
         resizable: false,
-        position: { my: "left top+10", at: "left bottom", of: wordElement, collision: "flip" },
+        position: wordElement
+                  ? { my: "left top+10", at: "left bottom", of: wordElement, collision: "flip" }
+                  : undefined, // no srcElement from the evt argument, use undefined to position in screen center
```

Comment:
If we had a `bounds` field as described above, then there's probably a corresponding solution to the positioning problem here.